typings/vscode-mssql.d.ts

```diff
@@ -76,6 +76,13 @@ declare module 'vscode-mssql' {
          */
         getDatabaseNameFromTreeNode(node: ITreeNodeInfo): string;
 
+        /**
+         * Get the connection string for the provided connection Uri
+         * @param connectionUri The URI of the connection to get the connection string for.
+         * @param includePassword to include password with connection string
+         * @returns connection string
+         */
+        getConnectionString(connectionUri: String, includePassword: boolean): Promise<string>;
```

Comment:
includePassword should be optional per the params typing (and update the comment to mention default is false)